cmd/minishift/cmd/addon/util.go

```diff
@@ -41,14 +41,14 @@ func GetAddOnManager() *manager.AddOnManager {
 func WriteAddOnConfig(addOnConfigMap map[string]*addon.AddOnConfig) {
 	c, err := config.ReadConfig()
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Unable to read the Minishift configuration: %s", err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Cannot read the Minishift configuration: %s", err.Error()))
```

Comment:
This is for consistency and conciseness. "Unable to" is considered in some parts more "high language" and "Cannot" is more universally understood by non-native English speakers. We also use "cannot" already in other commands, so I'm aligning the terminology and phrasing.